src/sentry/api/endpoints/project_app_store_connect_credentials.py

```diff
@@ -601,35 +486,19 @@ def post(self, request, project):
             return Response(status=404)
 
         serializer = AppStoreConnectRequestSmsSerializer(data=request.data)
-
         if not serializer.is_valid():
             return Response(serializer.errors, status=400)
+        data = serializer.validated_data["sessionContext"]
+        logger.warn("THE DATA:::::::::::::::::: %r", data)
```

Comment:
wouldn’t this put sensitive credentials into the log? I guess you meant this to be just temporary?

Reply:
ah yes.  good spot!